### ISSUE TITLE
Option for data splitting functions in 3 subsets for training, testing and validation

### ISSUE THREAD

**Issue by jmaspons** [topepo/caret#1222]
Lever, J., Krzywinski, M., & Altman, N. (2016). Model selection and overfitting. Nature Methods, 13(9), 703–704. https://doi.org/10.1038/nmeth.3968

**Comment by topepo** [comment]
No. After you make the train/test split, you can run `train()` with the control options `method = "LGOCV"` and `number = 1` to use a validation set.
